File: example/src/main/java/ru/virgil/example/image/ImageMockService.java
package ru.virgil.example.image;

import org.springframework.stereotype.Service;
import ru.virgil.example.user.UserDetails;
import ru.virgil.utils.FakerUtils;
import ru.virgil.utils.image.ImageService;

@Service
public class ImageMockService extends ru.virgil.utils.image.ImageMockService<UserDetails, PrivateImageFile> {

    public ImageMockService(ImageService<UserDetails, PrivateImageFile> imageService, FakerUtils fakerUtils) {
        super(imageService, fakerUtils);
    }
}

File: example/src/main/java/ru/virgil/example/image/ImageMockService.kt
package ru.virgil.example.image

import org.springframework.stereotype.Service
import ru.virgil.example.user.UserDetails
import ru.virgil.utils.Faker
import ru.virgil.utils.image.ImageMockService
import ru.virgil.utils.image.ImageProperties
import ru.virgil.utils.image.ImageService

@Service
class ImageMockService(
    imageService: ImageService<UserDetails, PrivateImageFile>,
    faker: Faker,
    imageProperties: ImageProperties,
) : ImageMockService<UserDetails, PrivateImageFile>(imageService, faker, imageProperties)
